### ISSUE TITLE
topdown inference + flow tracker fails to return instance predictions

### ISSUE THREAD

**Issue by ntabris** [murthylab/sleap#343]
When running topdown inference with the flow tracker, the inference results are completely off (either no points or very few points at random locations). I believe there's a bug in how we're retaining the full_image (for the flow tracker). I suspect the centered instance model is getting the wrong image but haven't confirmed this.

**Comment by ntabris** [comment]
example for testing:
```
sleap-track  D:/bee_video_clip.recoded.mp4  --frames  500-510  -m  D:/yan_bees\models\centroid200414_174657\initial_config.json  -m  D:/yan_bees\models\topdown200415_003856\training_config.json  --tracking.tracker  flow  --tracking.similarity  instance  --tracking.matching  greedy  --tracking.window  5  -o  D:/yan_bees\predictions\bee_video_clip.recoded.mp4.200508_094400.predictions.slp
```
(files are local on my Windows machine)

**Comment by ntabris** [comment]
One possible quick fix is to just drop the full_image from the example (like we do when not using flow tracker) and then reload when needed for tracker.

**Comment by ntabris** [comment]
Fixed in 4ab48be, deployed in v1.0.1.
